semantic-conventions/syntax.md

```diff
@@ -119,6 +119,18 @@ events ::= id {id} # MUST point to an existing event group
 
 name ::= string
 
+instrument ::=  "Counter" 
```

Comment:
Please also document the semantics of the new syntax (below sections).

Reply:
Done 👍